packages/server/billing/helpers/adjustUserCount.ts

```diff
@@ -83,7 +77,16 @@ const addUser = async (orgIds: string[], user: IUser, dataLoader: DataLoaderWork
   await getKysely()
     .insertInto('OrganizationUser')
     .values(docs)
-    .onConflict((oc) => oc.doNothing())
+    .onConflict((oc) =>
+      oc.constraint('unique_org_user').doUpdateSet({
+        joinedAt: sql`CURRENT_TIMESTAMP`,
+        removedAt: null,
+        inactive: false,
+        role: null,
+        suggestedTier: null,
+        tier: (eb) => eb.ref('excluded.tier')
+      })
+    )
```

Comment:
I'm mostly concerned with this part. Previously we just created a new org user when a new user joined, but now we have to reset all the old fields that should be reset.

Reply:
LGTM! i don't think we use trialStartDate anymore so we can ignore that column.